main/midnight/src/tme/scenario/item_character.cpp

```diff
@@ -374,10 +374,22 @@ namespace tme {
                 // TODO: recruiting logic
                 return true ;
             }else{
-                if(mx->isRuleEnabled(RF_LOM_UNRECRUITABLE_FEY)
-                    && character->Race() == RA_FEY)
-                {
-                    return false;
+                if ( character->Race() == RA_FEY) {
+                    if (mx->isRuleEnabled(RF_LOM_FEY_RECRUIT_OFF))
+                    {
+                        return false;
+                    }else if (mx->isRuleEnabled(RF_LOM_FEY_RECRUIT_NOVEL)) {
+                        // In the novel, Feys only join the cause after 
+                        // the Lord of Dreams joins.
+                        // The Lord of Dreams is required for recruiting Fey, 
+                        // he can always be recruited and other Fey can only
+                        // be recruited if he has already been recruited
+                        return character->Id() == mx->scenario->dreams->Id() ||
+                            mx->scenario->dreams->IsRecruited();
+                    }
+                    else {
+                        return true;
```

Comment:
@fassisrosa 

This shouldn't be here. It essentially means that all fey can be recruited in normal game mode rather than dropping down to the recruitment check underneath.

FYI: I merged this ticket in to a new branch and created a [draft PR ](https://github.com/ChilliHugger/The-Lords-Of-Midnight/pull/228)- so that I could run the tests and do a little housekeeping (my side) while the options code was being touched.

The running of the tests hilighted the bug.

The next is just a conversational piece...

I also wondered about the ACs - with RF_LOM_FEY_RECRUIT_NOVEL on, then dreams can be recruited by anyone - whereas normally (apart from other fey) he would only be recruited by MORKIN, CORLETH, or RORTHRON.

Once dreams is recruited, then ALL fey can be recruited by anyone - rather than again just by other fey and main 4. This kinda makes sense as far as the the novel option is concerned.

However, both could be modified by difficulty.